mvc/models/article.model.js

```diff
@@ -6,14 +6,34 @@ exports.selectArticleById = (articleId) => {
   return errorHandleDBQuery(query, [articleId], errorMsg).then((rows) => rows[0]);
 };
 
-exports.selectArticles = () => {
-  const query = `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.article_id) AS comment_count 
-  FROM articles
-  LEFT JOIN comments ON comments.article_id = articles.article_id
-  GROUP BY articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url
-  ORDER BY articles.created_at DESC;`;
-  const errorMsg = "";
-  return errorHandleDBQuery(query, null, errorMsg).then((rows) => rows);
+exports.selectArticles = (query) => {
+  const acceptedQuery = ["topic", "author", "sort_by", "order"];
```

Comment:
There should be a way of checking if the topic exists so we know the reason why if we get an empty array back (The existence of the topic should come from the db, not to be hard coded so it remains up to date)